.jules/sentinel.md

```
@@ -0,0 +1,4 @@
+## 2025-03-09 - XXE Vulnerability Mitigation
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)